code/script.js

```diff
@@ -34,18 +43,282 @@ const showMessage = (message, sender) => {
 }
 
 // Starts here
+// Bot greets user
 const greetUser = () => {
   // here we call the function showMessage, that we declared earlier with the argument "Hello there, What's your name?" for message, and the argument "bot" for sender
-  showMessage("Hello there, What's your name?", 'bot')
+  showMessage("Hi! I can help you find the perfect gift for anyone. <br><br>Please start by telling me your name.", 'bot')
```

Comment:
Adding <br><br> in the text is a great addition, it makes the text in the bubble nice and readable!